src/structures/ThreadChannel.js

```diff
@@ -253,6 +253,17 @@ class ThreadChannel extends Channel {
     return members.get(this.ownerId) ?? null;
   }
 
+  /**
+   * Fetches the message that started this thread, if any.
+   * <info>This only works when the thread started from a message in the parent channel, otherwise the promise will
+   * reject. If you just need the id of that message, use {@link ThreadChannel#id} instead.</info>
+   * @param {BaseFetchOptions} [options] Additional options for this fetch
+   * @returns {Promise<Message>}
+   */
+  fetchStarterMessage(options) {
+    return this.channel.messages.fetch(this.id, options);
```

Comment:
```suggestion
    return this.channel.messages.fetch(this.id, options) ?? null;
```

Reply:
Even if fetch didn’t throw an error this is still invalid

Reply:
this is not valid, `MessageManager#fetch` (`MessageManager#_fetchId`) does not return null